packages/web-react/src/types/tabs.ts

```diff
@@ -16,12 +24,14 @@ export interface SpiritTabsProps {
   id?: TabId;
   /** Identification of affected pane */
   forTabPane?: TabId;
+  spacing?: SpaceToken | Partial<Record<BreakpointToken, SpaceToken>>;
```

Comment:
You can keep this DRY: define a new interface `SpacingProps` and use it for extension the same way as `ChildrenProps` or `TransferProps`. 💡

Reply:
Unfortunately, SpacingProps already exist, so it's TabsSpacingProps. However, it might be best to use it shared for Stack and other components as well. But I can't really think of a name for it

Reply:
EDIT: refactored - also for Stack component